### PR TITLE
feat: Initialize project with MongoDB integration

### PR DESCRIPTION
**Pull request by mrbin264**
This pull request initializes a new NestJS project with MongoDB integration.

**Key changes:**

- Added MongoDB as the database and integrated it using Mongoose.
- Set up basic CRUD operations for interacting with the database.
- Included necessary configuration files and updated module dependencies.
- Added Docker Compose and Dockerfile for easy setup and deployment.
- Configured ESLint for TypeScript and added a pre-commit hook with Husky for code quality.
- Updated the README with an introduction to the Telemedicine Platform.

### CODE REVIEW

File: startup.document.dev.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# 2. Displays the contents of the .env file.
+# 3. Runs the database seed scripts.
+# 4. Starts the application in production mode.
+#!/usr/bin/env bash
+set -e
+
+/opt/wait-for-it.sh mongo:27017
+cat .env
+npm run seed:run:document
+npm run start:prod
```
Comment (vuquangit) [inline startup.document.dev.sh]: using pnpm ?

Comment (mrbin264) [reply startup.document.dev.sh]: Fixed 👯‍♂️

**Review by tuev — APPROVED** [review]
LGTM

**Review by vuquangit — APPROVED** [review]

**Review by trinhdinhhiep — APPROVED** [review]

**Comment by tuev** [comment]
@CodiumAI-Agent /describe

**Comment by tuev** [comment]
@CodiumAI-Agent /review

**Comment by tuev** [comment]
@CodiumAI-Agent /improve

**Pull request merged**